nesta/production/orms/meetup_es_config.json

```diff
@@ -45,13 +48,13 @@
                 "id_country_group": {
                     "type": "keyword"
                 },
-                "id_iso2_group": {
```

Comment:
I think `id_iso2_group`, `id_iso3_group` and `id_isoNumeric_group` were supposed to be correct while for CB and NIH we were supposed to do:

- `id_iso2_country` -> `id_iso2_organisation`
- `id_iso3_country` -> `id_iso3_organisation`
- `id_isoNumeric_country` -> `id_isoNumeric_organisation`

See (*)

Reply:
> * a field like `language_iso2_group` / `language_iso2_organisation` or so (if mentioning the entity type  as the last chunk is not mandatory, I'd use a single identifier like `id_iso2_language`)

A note: because of the implementation, this will be `terms_iso2lang_group` since there could be multiple languages in a piece of text.

Reply:
we settled on `*_iso*_country` over `*_iso*_<entity>` since our graphs will have countries as entities so, it suits future development to have the suffix as `country`

Reply:
OK. If we're going in this direction, looking at [aliases](https://github.com/nestauk/nesta/blob/dev/nesta/production/schemas/tier_1/aliases/health_scanner.json) why not do the same for:
- `continent_id` (`id_continent_organisation` -> `id_iso2_continent`)
- `countries_ids` (`terms_of_countryTags` -> `terms_iso2_countryTags`)
- `state_id` (`id_state_*` -> `id_iso2_state`)

Adn btw if we relied on the lookup only these could be removed completely, to reduce the response size:
- `continent`
- `country`
- `state`

Also, as languages could be graph nodes too, why not using `terms_iso2_lang` instead of `terms_iso2lang_group`?

Reply:
ok, agreed, however if we go along these lines then the best strategy would be to holistically look at everything and rewrite the mappings. 

Opened issue #124